src/assets/stylesheets/custom.scss

```diff
@@ -59,32 +53,32 @@ $font20: 20px;
 // custom classes
 .width100{ width: $width100; }
 .width50{ width: $width100/2; }
-.marginT20{ margin-top: $margin20; }
-.marginB10{ margin-bottom: $margin10; }
-.marginB20{ margin-bottom: $margin20; }
-.marginT5{ margin-top: $margin5; }
-.marginB5{ margin-bottom: $margin5; }
-.marginL5{ margin-left: $margin5; }
-.marginR5{ margin-right: $margin5; }
-.marginT10{ margin-top: $margin10; }
-.marginB10{ margin-bottom: $margin10; }
-.marginL10{ margin-left: $margin10; }
-.marginR10{ margin-right: $margin10; }
-.margin0{ margin: $margin0; }
-.paddingT20{ padding-top: $pad20; }
-.paddingB20{ padding-bottom: $pad20; }
-.paddingL20{ padding-left: $pad20; }
-.paddingR20{ padding-right: $pad20; }
-.paddingT5{ padding-top: $pad5; }
-.paddingB5{ padding-bottom: $pad5; }
-.paddingL5{ padding-left: $pad5; }
-.paddingR5{ padding-right: $pad5; }
-.padding0{ padding: $pad0; }
-.padding5{ padding: $pad5; }
-.padding20{ padding: $pad20; }
-.font16{ font-size: $font16; }
-.font18{ font-size: $font18; }
+.marginT20{ margin-top: em(20); }
```

Comment:
All of this is a bit confusing - why so many variables to remember? I feel like applying calculations to a base site would be more appropriate than having 22 different classes.